source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+    |                                                | Automatically generate masks binding tasks |
+    |                                                | to threads.                                |
+    +------------------------------------------------+--------------------------------------------+
+    | ``--threads-per-core=<threads>``               | In task layout, use the specified maximum  |
+    |                                                | number of hardware threads per core        |
+    |                                                | (default is 1; there are 2 hardware        |
+    |                                                | threads per physical CPU core).            |
+    |                                                | Must also be set in ``salloc`` or          |
+    |                                                | ``sbatch`` if using                        |
+    |                                                | ``--threads-per-core=2`` in your ``srun``  |
+    |                                                | command.                                   |
+    +------------------------------------------------+--------------------------------------------+
+    | ``-m, --distribution=<value>:<value>:<value>`` | Specifies the distribution of MPI ranks    |
+    |                                                | across compute nodes, sockets (L3          |
+    |                                                | regions), and cores, respectively.         |
+    |                                                | The default values are                     |
+    |                                                | ``block:cyclic:cyclic``, see ``man srun``  |
+    |                                                | for more information.                      |
+    |                                                | Currently, the distribution setting for    |
+    |                                                | cores (the third "<value>" entry) has no   |
+    |                                                | effect on Frontier                         |
+    +------------------------------------------------+--------------------------------------------+
+    |  ``--ntasks-per-node=<ntasks>``                | If used without ``-n``: requests that a    |
+    |                                                | specific number of tasks be invoked on     |
+    |                                                | each node.                                 |
+    |                                                | If used with ``-n``: treated as a          |
+    |                                                | *maximum* count of tasks per node.         |
+    +------------------------------------------------+--------------------------------------------+
+
+********
+Software
+********
+
+Gaea has several software and libraries available.  These are typically
```

Comment:
Too many availables. Maybe:

Gaea offers several software libraries.  These are typically
accessed using the :ref:`Lmod module system <gaea-environment-modules>` to
make the software available.  Use the `module avail` and `module spider`
commands to see the available software.

Reply:
Updated in a new push.